File: CineAPI/Models/Album.cs
namespace CineAPI.Models
{
	public class Album
	{
		public int Id { get; set; }

		public string Titulo { get; set; } = string.Empty;

		public string Artista { get; set; } = string.Empty;

		public int AnioLanzamiento { get; set; }

		public string Disquera { get; set; } = string.Empty;

		public int GeneroId { get; set; }

		public Genero? Genero { get; set; }
		public string Descripcion { get; set; } = string.Empty;
		public decimal Precio { get; set; }
		public string Imagen { get; set; } = string.Empty;
	}
}

File: musica-web/src/app/models/album.ts
import { Genero } from './genero';

export interface Album {
  id: number;
  titulo: string;
  artista: string;
  anioLanzamiento: number;
  disquera: string;
  descripcion?: string;
  precio?: number;
  imagen?: string;
  generoId: number;
  genero?: Genero;
}
